install_my_pihole.sh

```diff
@@ -1 +1,25 @@
 #!/bin/bash
+
+## CONSTANTS
+
+## FUNCTIONS
+function confirmName() {
+echo "So your full name is $1 $2?"
+}
+
+## GLOBAL VARS
+FIRST_NAME=""
+LAST_NAME=""
+
+## LOGIC
+echo "What first name?";
+read;
+FIRST_NAME=${REPLY}
```

Comment:
Is it possible to pass the input directly to the function, assigning to a var seems redundant.